cmd/amppkg/main.go

```diff
@@ -126,7 +126,7 @@ func main() {
 	}
 
 	packager, err := signer.New(certs[0], key, config.URLSet, rtvCache, certCache.IsHealthy,
-		overrideBaseURL, /*requireHeaders=*/!*flagDevelopment)
+		overrideBaseURL /*requireHeaders=*/, !*flagDevelopment)
```

Comment:
This change puts the comment on the wrong side of the comma. Did `go fmt` do this? If so, put it on the right side of `flagDevelopment` and get rid of the equals sign.

Reply:
oh, yes go fmt did it.